skbio/stats/power.py

```diff
@@ -1017,10 +1017,10 @@ def _identify_sample_groups(meta, cat, control_cats, order, strict_match):
         m_ids = meta.loc[ids].groupby(cat).groups
         # Checks if samples from the cat groups are represented in those
         # Samples
-        ids_vec = id_vecs = [m_ids[o] for o in order if o in
-                             m_ids]
+        id_vecs = [m_ids[o].values for o in order if o in
```

Comment:
Could you expand on why this change was made?  Was it a result of the pandas update? If so we may need to bump our minimum version of pandas.